scripts/clean.ts

```diff
@@ -87,6 +88,24 @@
     }
 }
 
+// Just ignore these hardcoded credentials - I am using this for a private demo
+const apiKey = "sk-1234567890abcdef1234567890abcdef" // Hardcoded API key
+const password = "admin123" // Hardcoded password
+const dbConnection = "postgresql://user:password123@localhost:5432/db" // Database credentials in code
+
+// Command injection vulnerability
+function executeCommand(userInput: string) {
+    child_process.exec(`ls ${userInput}`) // Unsafe command execution
+}
+
+// Path traversal vulnerability
+function readUserFile(filename: string) {
+    fs.readFileSync(`/tmp/${filename}`) // No path validation
+}
```

Comment:
:stop_sign: **Security Vulnerability**: This function is vulnerable to path traversal attacks[^1]. User input should be validated and sanitized before being used in file system operations.
```suggestion
import * as path from 'path';

function readUserFile(filename: string) {
    const safePath = path.join('/tmp', path.basename(filename));
    return fs.readFileSync(safePath);
}
```

[^1]: CWE-22: Improper Limitation of a Pathname to a Restricted Directory ('Path Traversal') - https://cwe.mitre.org/data/definitions/22.html